doc/api/os.md

```diff
@@ -19,9 +19,19 @@ system.
 added: v0.5.0
 -->
 
+    Stability: 0 - Deprecated: Use [`process.arch`][] instead.
+
 Returns the operating system CPU architecture. Possible values are `'x64'`,
 `'arm'` and `'ia32'`. Returns the value of [`process.arch`][].
 
+## os.BYTEORDER
```

Comment:
the reason for this now is just to clean out some long standing todo's I've had on my list. I tend to consider it a very small bug that this was defined as a function as opposed to a property.